Luna_Streaming_2.8_Release_Notes.md

```diff
@@ -51,6 +53,39 @@ If you are upgrading from Apache Pulsar 2.7.0 or Luna Streaming 2.7.2 you may ne
 This is because there is a breaking API change in Apache Pulsar 2.8.0 (and so in Luna Streaming 2.8.0) related to the SchemaInfo java class.
 More context [here](https://github.com/apache/pulsar/issues/11338).
 
+## Luna Streaming Distribution 2.8.0 1.1.5
+
+This release contains some bugfixes and a couple of relevant improvements, most notably:
+- New Debezium Oracle source
+- [Pulsar Proxy extensions](https://github.com/apache/pulsar/pull/11838) (from Apache Pulsar 2.9.0)
```

Comment:
this is a better link
https://github.com/apache/pulsar/wiki/PIP-99%3A-Pulsar-Proxy-Extensions